styles/alert.css

```diff
@@ -0,0 +1,64 @@
+    /*
+        Author: Vrushabh Dhatrak
+        Release: 0.1
+        Description : Alert Component Css
+    */
+    /* Alert Starting Here*/
+    .alert {
+        padding: 1rem;
+        border-radius: 0.5rem;
+        max-width: max-content;
+        box-sizing: border-box;
+        display: flex;
+        align-items: center;
+        justify-content: space-evenly;
+    }
+    .alert svg {
```

Comment:
Leave a line break b/w two definitions.

Reply:
Ok Thanx will do

Reply:
Let's use reactions for acknowledgements and replies for discussions only to avoid unnecessary notifications to the people involved in this PR. :)